README.md

```diff
@@ -90,23 +104,17 @@ Typical output:
 [   11.211998] [drm] Initialized intel_vpu 1.0.0 20230117 for 0000:00:0b.0 on minor 0
 ```
 
-Note in this output that the system initially boots with the firmware that ships with OS before reloading more recent firmware provided by the snap. Check the [upstream repo from Intel](https://github.com/intel/linux-npu-driver/releases) for the expected firmware version for your platform.
+In this example output the system initially boots with the firmware that ships with OS before reloading more recent firmware provided by the snap. Check the [upstream repo from Intel](https://github.com/intel/linux-npu-driver/releases) for the expected firmware version for your platform.
 
 ### Running the vpu-umd-test application
 
-First connect to the `custom-device` interface, which allows access to the NPU device node on the host:
-
-```
-sudo snap connect intel-npu-driver:intel-npu-plug intel-npu-driver:intel-npu
-```
-
-If you have not done so already, ensure the following are performed in order to set up non-root access to the NPU device:
+To allow non-root access to the NPU device, first ensure the appropriate user is in the `render` Unix group:
 
 ```
 sudo usermod -a -G render $USER # log out and log back in
 ```
 
-If this is your first run, or if you re-loaded the `intel_vpu` driver, then you'll also need to perform the following:
+Next apply the appropriate permissions on the device node. This is required each time the `intel_vpu` driver is reloaded, e.g. when the snap is first installed or following a reboot.
 
 ```
 sudo chown root:render /dev/accel/accel0
```

Comment:
The `usermod` and `chown` instructions are necessary only if the user wants to run `vpu-umd-test` in no-root mode right ?

Reply:
@hector-cao Yes, that's correct.

Reply:
by curiosity, what are the default ownership/permissions settings of `/dev/accel/accelxx` devices ?

Reply:
```
$ ls -lh /dev/accel/accel0
crw------- 1 root root 261, 0 Oct 16 15:27 /dev/accel/accel0
```